docs/events.md

```diff
@@ -25,3 +25,16 @@ AddEventHandler('txAdmin:events:scheduledRestart', function(eventData)
     end
 end)
 ```
+
+## txAdmin:events:broadcast
+Called for whenever an announcement is triggered from the admin panel.  
+Arguments:
+- `user`: Username of the person triggering the announcement.  
+- `announcement`: Announcement text
+
+Example usage on ESX v1.2:
```

Comment:
this code doesn't depend on esx at all

Reply:
Whoops, copy pasted this from the other event documentation, forgot to remove it lol.